src/main/java/bridge/domain/BridgeGame.java

```diff
@@ -0,0 +1,113 @@
+package bridge.domain;
+
+/**
+ * 다리 건너기 게임을 관리하는 클래스
+ */
+public class BridgeGame {
+
+    private final AnswerBridge answerBridge;
+    private PlayerBridge playerBridge;
+    private boolean endGame;
+    private final GameCount gameCount;
+
+    public BridgeGame(AnswerBridge answerBridge, PlayerBridge playerBridge, boolean endGame, GameCount gameCount) {
+        this.answerBridge = answerBridge;
+        this.playerBridge = playerBridge;
+        this.endGame = endGame;
+        this.gameCount = gameCount;
+    }
+
+    public static BridgeGame initGame(AnswerBridge answerBridge) { // 여기 분리
+        PlayerBridge playerBridge = PlayerBridge.init();
+        boolean isSuccess = false;
+        GameCount gameCount = GameCount.init();
+        return new BridgeGame(answerBridge, playerBridge, isSuccess, gameCount);
+    }
```

Comment:
(개인적인 생각입니다.)

```suggestion
    public static BridgeGame initGame(AnswerBridge answerBridge) { // 여기 분리
        return new BridgeGame(answerBridge, PlayerBridge.init(), false, GameCount.init());
    }
```

로컬 변수가 정말 필요한지 고민해보면 좋을 것 같아요.
포비의 TDD 세미나에서 얻은 인사이트인데, 재활용하는 곳이 없으면 의식저으로 줄여보는 연습을 하는 것도 깔끔한 코드를 작성하는데에 큰 도움을 주더라고요 :)